From eb4cedf1097bb556134a03648a99c60b16fa4726 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jan=20=C4=8Cern=C3=BD?= <jcerny@redhat.com>
Date: Tue, 5 Dec 2023 16:22:29 +0100
Subject: [PATCH 12/14] Add SRG id to `file_owner_grub2_cfg` for RHEL 9 STIG

Patch-name: scap-security-guide-0.1.71-add_srg_to_file_owner_grub2_cfg-PR_11261.patch
Patch-status: Add SRG id to `file_owner_grub2_cfg` for RHEL 9 STIG
---
 .../bootloader-grub2/non-uefi/file_owner_grub2_cfg/rule.yml      | 1 +
 1 file changed, 1 insertion(+)

diff --git a/linux_os/guide/system/bootloader-grub2/non-uefi/file_owner_grub2_cfg/rule.yml b/linux_os/guide/system/bootloader-grub2/non-uefi/file_owner_grub2_cfg/rule.yml
index fef91a47df..3df07a5689 100644
--- a/linux_os/guide/system/bootloader-grub2/non-uefi/file_owner_grub2_cfg/rule.yml
+++ b/linux_os/guide/system/bootloader-grub2/non-uefi/file_owner_grub2_cfg/rule.yml
@@ -45,6 +45,7 @@ references:
     nist-csf: PR.AC-4,PR.DS-5
     pcidss: Req-7.1
     pcidss4: "2.2.6"
+    srg: SRG-OS-000480-GPOS-00227
     stigid@rhel9: RHEL-09-212030
 
 ocil_clause: '{{{ ocil_clause_file_owner(file=grub2_boot_path ~ "/grub.cfg", owner="root") }}}'
-- 
2.43.0

